src/Dto/RegisterResponse.php

```diff
@@ -57,7 +57,7 @@
      * @var int
      * @see https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-javascript-api.html#error-codes
      */
-    public $errorCode;
+    public $errorCode = self::ERROR_CODE_OK;
```

Comment:
Question: why use the "not used but reserved error code"? Also, why instantiate with "success" by default as that seems to be something that should be avoided (you fail by default unless we're sure it is successful, then it passes...). Same with the `SignResponse` btw.